resources/views/vendor/filament-panels/components/login-version-card.blade.php

```diff
@@ -0,0 +1,43 @@
+{{--
+<COPYRIGHT>
+
+    Copyright © 2016-2024, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+--}}
+<div class="bg-turkish-500 text-white rounded-xl p-6 mb-6">
+    <div class="flex md:flex-row flex-col justify-center items-center gap-6">
+        <div>
+            <p class="text-white text-sm font-semibold">Version {{ app('current-version') }} is now available!</p>
+            <p class="text-white-500 text-sm my-4">Your instance of Advising App<sup>TM</sup> was automatically updated with our latest available features.</p>
```

Comment:
Though it shows it in the mockups, we actually just recently became a registered trademark. So let's change the TM to the official registered trademark icon `&#174;`